### ISSUE TITLE
Supernote A5X Support

### ISSUE THREAD

**Issue by chrisbopp** [klimeryk/recalendar.js#14]
This tool is amazing, it's exactly what I need! Thank you so much for your work on this!

I saw you mentioned that you only designed the tool for the ReMarkable since that's the device you have, but I wondered if you would consider supporting the [Supernote A5X](https://supernote.com). I believe that the only modification would be to make space at the top for the toolbar.

Based on [this reddit post](https://reddit.com/r/Supernote/comments/rur7z4/) it seems it would need about a 100 px blank space at the top of every page. Given that loss of space, it also would be amazing if the top part of the planner could be condensed a bit, but that's more of a nice-to-have feature.

The A5X is a 10.3-inch E Ink display with a 1404×1872 resolution at 226 PPI. The top menu bar that I mentioned is pictured in section 2.3 in the [user manual](https://prod-ratta-cms.s3.ap-northeast-1.amazonaws.com/V2.1/PDF/User's+Manual_X+V2.1.pdf).

That reddit post also indicates some loss of pixels on the left and right as well, but I'm not sure that's enough to matter. I do not yet have my Supernote in hand (waiting on delivery), but should have it soon to test on. Thanks!

**Comment by klimeryk** [comment]
Thanks for starting the issue, @chrisbopp! I'll definitely have a look this week. I'll try to generate a test PDF to meet those requirements and share it here. I hope you'll be wiling to test it out on your device to make sure it's working well (and I'll iterate if needed). Once we figure out the optimal settings, I'll add it as a proper, selectable option on the webapp.

**Comment by chrisbopp** [comment]
Excellent, thanks @klimeryk! I can definitely test on my device, but I’ve been told it probably won’t arrive until next week so no rush. Thanks again for this awesome resource!

**Comment by RichardFoxworthy** [comment]
Hi @klimeryk and @chrisbopp , I came looking for exactly this! A Supernote A5X option for this planner template would be fantastic! I do already have an A5X and can help with testing any prototypes if that's useful.

The existing ReMarkable format is almost usable on my A5X, but as @chrisbopp noted, the header area gets cut off by the Supernote menu bar. All the hyperlinks work fine.

**Comment by klimeryk** [comment]
Related issue/comment: https://github.com/klimeryk/recalendar.js/issues/123#issuecomment-2439751122. I've just added support for ReMarkable Paper Pro, which required adding logic for custom resolutions and DPIs.
Upon further investigation, looks like the screen specification (resolution and DPI) are exactly the same as ReMarkable 1&2. So I've just added an explicit mention of Supernote A5 X in the Device dropdown, so that folks know it's compatible/works.
Based on [this comment](https://github.com/klimeryk/recalendar.js/pull/42#issuecomment-1311717019) from @sinistersnare, looks like maybe the top margin is not needed?
Otherwise, if someone could provide a screenshot/photo of how that looks like, ideally with ReCalendar loaded, that'd help me understand the issue better and how to tackle it 🙏
